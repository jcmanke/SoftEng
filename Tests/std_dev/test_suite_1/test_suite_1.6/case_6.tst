1543.14
63.124
145.124
161.423
13.56
63451.5324
543.1254
1.654
234.43
654.23
