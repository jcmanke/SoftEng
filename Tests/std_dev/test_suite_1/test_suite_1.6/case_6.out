The average score is: 6681.13
The Standard Deviation is: 19952.5
